docker-compose.yml

```diff
@@ -21,15 +21,17 @@ services:
 
   watcher:
     <<: *fastai
-    command: watchmedo shell-command --command nbdev_build_docs --pattern *.ipynb --recursive --drop
+    command: >
+            bash -c "pip install .
+            &&watchmedo shell-command --command nbagile_build_docs --pattern *.ipynb --recursive --drop"
     network_mode: host # for GitHub Codespaces https://github.com/features/codespaces/
 
   jekyll:
     <<: *fastai
     ports:
-     - "4000:4000"
+     - "4004:4000"
```

Comment:
Interesting that a different port map was needed. Did the port suddenly change?

Reply:
That's actually a good point, that probably isn't necessary.  I thought maybe I had a port conflict so I changed it to map to my external port 4004 instead of 4000 but that didn't end up being my problem.  It was the slashes that were causing issues.  I would be find reverting that back to 4000:4000

Reply:
Otherwise, if there are going to be multiple things mapping to port 4000 (thinking nbagile and nbverbose), is there something we could do to variablize that number instead so it would be configurable?  I've used .env for this in the past.